packages/itwinui-react/src/core/utils/hooks/useTheme.ts

```diff
@@ -6,8 +6,7 @@ import * as React from 'react';
 import { getDocument, getWindow } from '../functions/index.js';
 import { useIsomorphicLayoutEffect } from './useIsomorphicLayoutEffect.js';
 import { useIsThemeAlreadySet } from './useIsThemeAlreadySet.js';
-import '@itwin/itwinui-css/css/global.css';
```

Comment:
If we're not using the imports anymore, will they disappear from the repo? They're still being used by the html pages I assume.

Reply:
please read the PR description. this is the same change as #1322